src/header/css/header-apps.pcss

```diff
@@ -169,6 +169,10 @@ html[dir="rtl"] .dropdown-menu.dropdown-right {
 
 .esri-header-apps .dropdown-width-3 {
 	min-width: 480px;
+
+	@media (width <= 480px) {
+		min-width: 350px;
```

Comment:
So on really small phones (320px) this will side scroll, is that desired?

Reply:
No I don't think that's the desired behavior.  Do we add another media query for (width <= 350px) or is there a better way to make this work better overall?

Reply:
You could play with sizing to `100vw` or something (could also use the `--esir-vw` unit maybe?)